Dockerfile

```diff
@@ -1,3 +1,15 @@
 FROM nginx:alpine
-COPY ./www /usr/share/nginx/html
+
+# install required packages
+RUN apk add --no-cache git
+RUN apk add --update nodejs npm
```

Comment:
I'd put these in one RUN statement to reduce layers, once you're happy with it.
See [docker best practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run)